File: Angular/IELTS-PTE/src/app/admin/reading/fill-in-blank/filling-the-blank-answer-display/fillInBlanks.ts
export class FillInBlanks{
    qestion:RequestText[];
    answers:string[];
    timer:number;
  }

  export class RequestText{
    text:string;
    textType:number;
    requestAnswer:Answers;
  }

  export class Answers{
    id:number;
    correctAnswer:string;
    selectAnswer?:string;
    isRemove:boolean;
  }

  export class Question{
    id:number;
    questionName: string;
    questionDetails:QuestionSetup;
    fullQuestion: FillInBlanks;
  }

  export class Questions{
    questionList:Question[];
  }

  export class Answer{
    answerName:string;
  }

  export class QuestionSetup{
    id:number;
    subject:string;
    question:string;
    timer:number;
    isFree:boolean;
    status:boolean;
    answerList:string[];
    modalId?:string;
    targetId?:string;
  }

File: IELTS_PTE_API/IELTS_PTE.Common/ReturnEntities/Reading/fillInBlanks.cs
using System;
using System.Collections.Generic;
using System.Text;
using IELTS_PTE.Common.ReturnEntities.Reading;

namespace IELTS_PTE.Common.ReturnEntities
{
    public class FillInBlanks
    {
        public FillInBlanks()
        {
            qestion=new List<RequestText>();
            answers=new List<string>();
        }

        public List<RequestText> qestion { get; set; }
        public List<string> answers { get; set; }
        public int timer { get; set; }
    }
}
